src/gpb.erl

```diff
@@ -34,8 +34,8 @@
 -export([defs_records_to_proplists/1,  proplists_to_defs_records/1]).
 -export([rpc_records_to_proplists/1, rpc_record_to_proplist/1, proplists_to_rpc_records/1]).
 -include_lib("eunit/include/eunit.hrl").
--include("../include/gpb.hrl").
--include("../include/gpb_version.hrl").
+-include("gpb.hrl").
```

Comment:
Ah, ok. Might be worth knowing that the `-include` line in `gpb_parse.yrl` looked a little different from the rest (in case you're search-replacing automatedly or something, just so you won't miss it). It looked:
```erlang
-include("include/gpb.hrl").
```